tests/fixtures/windows-10/dir-C.out

```diff
@@ -0,0 +1,21 @@
+ Volume in drive C has no label.
+ Volume Serial Number is 246B-3B5A
+
+ Directory of C:\Program Files\Internet Explorer
```

Comment:
Might be interesting to pull `C:\Program Files\Internet Explorer` into a `parent_directory` field in each `output_line`.